packages/react/src/components/BarChartCard/__snapshots__/BarChartCard.story.storyshot

```diff
@@ -1753,10 +1753,10 @@ exports[`Storybook Snapshot tests and console checks Storyshots Watson IoT/BarCh
                     </div>
                     <button
                       className="bx--btn--icon-only iot--tooltip-svg-wrapper bx--btn bx--btn--ghost"
-                      data-testid="download-button"
                       disabled={false}
                       onClick={[Function]}
                       tabIndex={0}
+                      testID="download-button"
```

Comment:
this doesn't seem quite right

Reply:
Fixed!! Thanks for finding out the issue!